source/help/submit_status.md

```diff
@@ -57,17 +58,16 @@ remove it from this status, and return it to [incomplete](submit_status.md#incom
 On Hold
 -------
 
-Your submission was identified by arXiv staff or [moderators](moderation/index.md) as needing further attention.
+All submissions to arXiv go through a series of automated and human checks as [part of moderation](moderation/index.md). 
 
-Submissions may be put on hold for a variety of reasons, ranging from
-questions about proper classification, presentation issues, copyrighted PDF, or editorial concerns. Most
-of these do not require any further input from the submitter and will be
-dealt with in due course. arXiv urges submitters to be patient. Due to
-the large volume of submissions, it may take several days before a resolution is reached.
+When a submission goes on hold, arXiv editors will either send you an email with instructions on resolving the hold or resolve the hold without your needing to do anything. We will contact you using the email address in your user account with any changes to your submission status.
```

Comment:
Suggestion: "... arXiv editors will either send you an email with instructions on how to resolve the hold or we will resolve the hold for you.".